_includes/head.html

```diff
@@ -48,8 +48,14 @@
   {% else %}
   <meta property="og:title" content="SensorFleet" />
   <title>SensorFleet</title>
-  {% endif %} {% if page.description %}
+  {% endif %}
+  {% if page.description %}
   <meta name="description" content="{{ page.description }}" />
+  {% else %}
+  <meta
```

Comment:
Not a good idea according to internets:

"If all of your pages have similar meta descriptions, they will be more misleading than helpful.

If you don’t have time to write meta descriptions for every page, you can let Google generate them automatically. Based on the query, Google will pull relevant parts of your page’s content to feature in the snippet."

Reply:
That was there before, this modification just removes the duplicate.
You think we should remove the default description totally? What pages are affected, should we do that in another mr and leave the blog be?